tests/test_sql.py

```diff
@@ -336,6 +336,17 @@ def test_uses_variables_from_snowflake_yml(
     mock_execute_query.assert_called_once_with("select foo_value")
 
 
+@mock.patch("snowflake.cli.plugins.sql.commands.SqlManager._execute_string")
+def test_uses_variables_from_snowflake_local_yml(
+    mock_execute_query, project_directory, runner
+):
+    with project_directory("sql_templating"):
+        result = runner.invoke(["sql", "-q", "select &{ ctx.env.sf_var_override }"])
+
+    assert result.exit_code == 0
+    mock_execute_query.assert_called_once_with("select foo_value_override")
```

Comment:
It would be nice to check that sf_var is still defined too? Or is that covered elsewhere?